packages/react-lowcode/src/ide-integration/identifierLength.ts

```diff
@@ -0,0 +1,13 @@
+import {
+  SourceLineCol,
+  startOfJsxIdentifier,
+  astFindStart,
+} from "../ast/find"
+import { SourceFile } from "ts-morph"
+
+export const getJsxIdentifierLength = (code: string, source: SourceLineCol) => {
```

Comment:
please move: 
`ide-integration/identifierLength.ts` to `src/ast/identifierLength.ts`
`ide-integration/clone.ts` to `src/clone...`
`ide-integration/remove.ts` to `src/ast/remove.ts`
`ide-integration/format.ts` to `src/ast/format.ts`